src/internal/operators/bufferTime.ts

```diff
@@ -144,8 +146,14 @@ class BufferTimeSubscriber<T> extends Subscriber<T> {
       const timeSpanOnlyState = { subscriber: this, context, bufferTimeSpan };
       this.add(context.closeAction = scheduler.schedule(dispatchBufferTimeSpanOnly, bufferTimeSpan, timeSpanOnlyState));
     } else {
-      const closeState = { subscriber: this, context };
-      const creationState: DispatchCreateArg<T> = { bufferTimeSpan, bufferCreationInterval, subscriber: this, scheduler };
+      const closeState = { subscriber: this, context, destination: this.destination };
+      const creationState: DispatchCreateArg<T> = {
+        bufferTimeSpan,
+        bufferCreationInterval,
+        subscriber: this,
+        scheduler,
+        destination: this.destination
```

Comment:
Unfortunately, because of this pattern of passing explicit state into the scheduler's callback as an argument like this, it means we need to copy in the private member here, so it can "next" onto it in the close action. If there is a better way, let me know. My idea to improve this would be to close over state instead of passing it in.